index.html

```diff
@@ -225,6 +226,210 @@ <h1 class="playfair">Get Started</h1>
             </div>
         </footer>
     </section>
-</body>
 
+    <!--Desktop-Section-->
+    <section class="desktop">
+        <section class="desktop desktop-header">
+            <header>
+                <nav>
+                    <ul class="poppins">
+                        <li><a href="#">Hello</a></li>
+                        <li><a href="#">Portfolio</a></li>
+                        <li><a href="#">About</a></li>
+                        <li><a href="#">Contact</a></li>
+                    </ul>
+                </nav>
+            </header>
+        </section>
+    
+        <section class="desktop desktop-hello">
+            <div class="hello-text">
+                <h1 class="playfair">Okolo <br> Oshoke</h1>
+                <p class="poppins">Hello! I am a software developer! I can help you build a product, feature or website. Take a look of my works.
+                    If you like what you see and have a project you need coded, don&rsquo;t hesitate and contact me.
+                </p>
+                <div>
+                    <button type="button" class="poppins">Start Collaboration</button>
+                </div>
+            </div>                   
+        </section>
+    
+       <section class="desktop desktop-portfolio grid-2">
+            <div class="grid-2-1"><h1 class="playfair ">Projects</h1></div>
+            <div class="grid-2-2">
+                <img class="img3" src="img/projects-dots-1.svg" alt="">
+            </div>
+            <div class="port-entry-1 grid-2-3" >                
+                <div class="poppins">
+                    <h1>Project name goes here</h1>
+                    <ul>
+                        <li>HTML/CSS</li>
+                        <li>Ruby on Rails</li>
+                        <li>Javascript</li> 
```

Comment:
- [ ] **Required:** Just a minor change here 🔹 😃 ... Kindly update the language links color (seems like you used black) to match your [figma design](https://www.figma.com/file/l7SqJ3ZfkAKih9sFxvWSR4/Microverse-Student-Project-1?node-id=67%3A553) as per requirement 💪🏽 

## How your project looks 
![image](https://user-images.githubusercontent.com/35315311/178087574-5e74b4fa-2d28-4bc4-b4a3-70d7fbe34c8e.png)

# vs Figma design (#1A2236)
![image](https://user-images.githubusercontent.com/35315311/178087603-7c65e484-362a-4b37-be61-2ebf0f7d5a2c.png)